staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -150,6 +151,59 @@ var (
 		[]string{"requestKind"},
 	)
 
+	// low and high concurrency watermarks from the last few windows
+	concurrencyWatermarks = compbasemetrics.NewGaugeVec(
+		&compbasemetrics.GaugeOpts{
+			Name:           "apiserver_inflight_count_watermarks",
+			Help:           "Low and high watermark of mutating, readonly requests executing in recent windows.",
+			StabilityLevel: compbasemetrics.ALPHA,
+		},
+		[]string{"requestKind", "mark", "lag"},
```

Comment:
`requestKind` violates the instrumentation guidelines, everything should be snake_case